src/tests/ftest/pool/bad_exclude.py

```diff
@@ -125,7 +117,7 @@ def test_exclude(self):
         pool = None
         try:
             # setup the DAOS python API
-            with open('../../../.build_vars.json') as build_file:
+            with open('../../.build_vars.json') as build_file:
```

Comment:
Now that this file inherits `TestWithoutServers`, lines 120-122 can be removed and line 126 can use `self.context`:
```
            pool = DaosPool(self.context)
```